aerogear/mobile-developer-console
# add a new endpoint to return client config

To verify:

1. Make sure you have an openshift cluster running somewhere, and create a new project.
2. Login to the cluster using `oc`
3. Provision the keycloak operator using the templates provided in this PR:
    ```
    oc create -f ./deploy/services/keycloak/rbac.yaml
    oc create -f ./deploy/services/keycloak/operator.yaml
    oc create -f ./deploy/services/keycloak/install.yaml
    ```
4. You should see a keycloak instance will be provisioned.
5. Check the README file for setup the development environment. 
6. Go to the MDC and create a new app. Go to the services page and bind the app to IDM.
7. You should see the content of `mobile-services.json` file updated once the app is bound to the service.
